src/ksyms.cpp

```diff
@@ -78,7 +77,7 @@ std::optional<std::string> Ksyms::resolve_blazesym_int(uint64_t addr,
   uint64_t addrs[1] = { addr };
 
   const blaze_syms *syms = blaze_symbolize_kernel_abs_addrs(
-      symbolizer_, &src, addrs, ARRAY_SIZE(addrs));
+      symbolizer_, &src, addrs, 1);
```

Comment:
Why this change? I like that you don't have to remember to update this if `addrs` size changes

Reply:
It is within a function that accepts only a single parameter, and so I think this possibility adds even more confusion. Perhaps I should get rid of the array altogether and just use (&addr, 1)? That makes it even cleaner.

Just for context: the reason I made this change is that when homing things, I tried to delete unused things or home inside the implementation file if used only once (e.g. there were a couple declarations for functions that weren't even defined, didn't make sense to keep them, and some things that didn't need to be in the header). This #define didn't fit anywhere obvious, and given that was used only in one other place, and this use seemed simpler without, I updated this and homed it there rather than make a new header.

Reply:
I've made this `&addr, 1` to avoid the intermediate array. This probably matches best with the function here.

Just a note re: the addrs size changing, all the logic below is also predicated on this being implicitly size 1, so if you stuffed more addressed here (after changing the prototype, etc.) then you'd need to change that logic anyways.